### PR TITLE
Feature/infolistitem ng content

### LINKED ISSUE #41: InfoListItem title should support ng content
#### Describe the desired behavior
InfoListItem title should support ng content

#### Describe the current behavior
It only takes a string type input

#### Is this request related to a current issue?
No

### PR DESCRIPTION
**Pull request by emclaug2**
Fixes #41  #37 

Changes proposed in this Pull Request:
- Make title in InfoListItem ng-content
- Make subtitle in InfoListItem ng-content
- Rename `rightComponent` and `leftComponent` to `rightContent` and `leftContent`

### CODE REVIEW

File: docs/ScoreCard.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Child element with attributes:
 
 <div style="overflow: auto;">
 
-| Attributes  | Description                                 | Required | Default |
+| Selector    | Description                                 | Required | Default |
```
Comment (huayunh) [inline docs/ScoreCard.md:77]: May want to change everything

Comment (emclaug2) [reply docs/ScoreCard.md:77]: You're right, that was an accident change.  I think it makes sense to change `attributes` to `selector`, but not as a part of this PR.

**Comment by emclaug2** [comment]
> So can we not pass in title and subtitle as strings anymore?

I think <ng-content> is a catch-all when it comes to these kind of things.  
If someone passes in a `<div title>Title Here</div>` or a span, our typography is still applied to the element they pass in.

If it's a convenience thing, I can add back the string inputs.  It's just kind of confusing to me to have two entry points to the same thing in the component.

**Comment by joebochill** [comment]
> > So can we not pass in title and subtitle as strings anymore?
> 
> I think `<ng-content>` is a catch-all when it comes to these kind of things.
> If someone passes in a `<div title>Title Here</div>` or a span, our typography is still applied to the element they pass in.
> 
> If it's a convenience thing, I can add back the string inputs. It's just kind of confusing to me to have two entry points to the same thing in the component.

I think the markup for a user is cleaner with the attribute approach (since title will be used every time you use a ILI), but this is fine for now...we'll see if we get any pushback on it and we can add it back later if needed.

**Pull request merged**